imports/plugins/core/accounts/client/templates/profile/profile.js

```diff
@@ -88,6 +163,30 @@ Template.accountProfile.helpers({
     }
   },
 
+  /**
+   * @method displayEmail
+   * @summary returns a user's email.
+   * @since 1.5.0
+   * @return {String} - the email of a given user.
+   */
+  displayEmail() {
+    if (Reaction.Subscriptions && Reaction.Subscriptions.Account && Reaction.Subscriptions.Account.ready()) {
+      const account = getTargetAccount();
+
+      if (account && account.emails && account.emails[0]) {
+        return account.emails[0].address;
```

Comment:
I don't understand. Do you mean `account.emails[0].default`?

Reply:
@foladipo no, 
account.emails is an array. When we're getting an item out of an array we should be attempting to get the _correct_ item from the array rather than whichever item is in the first position. This is why emails have the `provides` property, and in most cases right now, the `"default"` email is the correct email to use. We should expect that users may have multiple email addresses, and that the email they add first may not be the email they are actively using or that they wish to use for this app.

If you see the code I wrote, it's looking for the email with `provides: "default" rather than just the first element in the array.

Reply:
Oh, I see. I didn't know about this use for the `provides` key in each element. And I do agree that using it as explained above does make more sense.